_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: default
+title: Mutate event
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Mutate event processors
+
+Mutate event processors allow you to modify events in Data Prepper. You can use these processors to add entries to an event, delete entries from an event, rename keys in an event, copy values within an event, and convert value types in an event. 
+
+The following processors allow you to mutate an event.
+
+## add_entries
```

Comment:
All processor names should follow the same convention. If it's the Java convention, this should be "AddEntries processor".  I'm not sure where the underscore is coming from: can this processor be used by referring to it as "add_entries"? Also, it only needs to be in code form if you can use it as a variable somewhere.

Reply:
It looks like this processor should be `Rename Key`, but I'll confirm the rest.